test/addons/async-hooks-promise/test.js

```diff
@@ -11,16 +11,18 @@ if (process.env.NODE_TEST_WITH_ASYNC_HOOKS) {
 }
 
 // Baseline to make sure the internal field isn't being set.
+// Because of bootstrap tracking this will be enabled.
 assert.strictEqual(
-  binding.getPromiseField(Promise.resolve(1)),
-  0);
+  binding.getPromiseField(Promise.resolve(1)).isChainedPromise,
+  undefined);
```

Comment:
I'm not sure exactly what part of the bootstrap refactoring has changed this test. The change is still true to the intent of the test, but it could be worth confirming the root cause.